Subscribe page.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask, request, render_template

```

Comment:
The subscribe page should be and addition to the current app, not another app with a subscribe page